filesystem/flysystem/AssetAdaptor.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace SilverStripe\Filesystem\Flysystem;
+
+use Controller;
+use Director;
+use League\Flysystem\Adapter\Local;
+
+/**
+ * Adaptor for local filesystem based on assets directory
+ */
+class AssetAdaptor extends Local {
```

Comment:
Adapt*o*r or adapt*e*r? I don’t even know which is correct, but I’d prefer if we were consistent with Flysystem’s naming (`use League\Flysystem\Adapter\Local`)